### DISCUSSION TITLE
如何在win下构建win版

### DISCUSSION THREAD

**Discussion by Hugo-Jiang** [opengnb/opengnb#36] [Q&A] [answered]
如题, 请求出一份详细的教程

**Comment by gnbdev** [comment]
把基于 Linux mingw 构建的 makefile 改一下就可以了，很简单的。
有用户已经出来成功编译，可我也不记得是哪位用户。
目前没有计划去写这类教程，恐怕你需要找熟悉这方面的朋友帮你。

**Reply by Hugo-Jiang** [reply]
好的, 感谢

**Comment by FanHeran** [comment]
https://github.com/gnbdev/gnb_build/tree/main/bin/Window10_x86_64

**Reply by FanHeran** [reply]
以上连接是编译好的，这个项目针对于win的Makefile文件有些问题，我自己修改了一下可以正常编译，你需要我可以提供

**Reply by Hugo-Jiang** [reply]
感谢回复,但这个应该是旧版本的. 我想说的是基于作者更新后的代码重新编译的版本

**Reply by Hugo-Jiang** [reply]
麻烦可以提供一下吗? 感谢

**Reply by FanHeran** [reply, marked as answer]
https://github.com/FanHeran/opengnb/releases

**Reply by Hugo-Jiang** [reply]
非常感谢!
